src/pages/api/arbeidsgiver/oppfolgingsplaner/[oppfolgingsplanId]/oppgave/[arbeidsoppgaveId]/slett.ts

```diff
@@ -0,0 +1,55 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { Sykmelding } from "../../../../../../../schema/sykmeldingSchema";
+import { isMockBackend } from "../../../../../../../environments/publicEnv";
+import { getSyfoOppfolgingsplanserviceTokenFromRequest } from "../../../../../../../server/auth/tokenx/getTokenXFromRequest";
+import {
+  getArbeidsoppgaveIdFromRequest,
+  getOppfolgingsplanIdFromRequest,
+} from "../../../../../../../server/utils/requestUtils";
+import { deleteOppgave } from "../../../../../../../server/service/oppfolgingsplanService";
+import { beskyttetApi } from "../../../../../../../server/auth/beskyttetApi";
+import getMockDb from "../../../../../../../server/data/mock/getMockDb";
+import {
+  ApiErrorException,
+  generalError,
+} from "../../../../../../../api/axios/errors";
+
+const handler = async (
```

Comment:
I `syfooppfolgingsplanservice` er de fleste av endepunktene felles for AG og SM, blant annet for å slette og lagre arbeidsoppgave. Jeg synes ikke vi trenger egne hooks/handlere for det i frontend, det kaller jo samme endepunkt, så jeg tenker at vi heller kan flytte de eksisterende handlerene/hooksene inn i en felles mappe. Jeg har gjort noe liknende i #95 for kopier-funksjonen.